packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.tsx

```diff
@@ -562,11 +571,15 @@ const VideoPlayerComponent: ForwardRefRenderFunction<HTMLVideoElement, VideoPlay
         }
       }
     }
-    if (!activeTrack) {
-      setActiveTrack(findDefaultLanguage || availableTextTracks[0])
-    }
   }, [activeTrack, availableTextTracks, captionsEnabled, captionsLanguage, player, storedLanguageExists])
 
+  useMountEffect(() => {
+    if (!captionsEnabled) {
```

Comment:
Well, you removed the condition from here but this doesn't really solve the issue. `findDefaultLanguage` has the same exact condition:
```ts
    if (!availableTextTracks) {
      return
    }
```
Is this mount effect needed at all? Based on the code I really doubt it has any effect, since like I said, `availableTextTracks` will most likely be always empty on first render